### DISCUSSION TITLE
When can we aspect a Chrome/ Firefox version?

### DISCUSSION THREAD

**Discussion by 0nhub** [mshibanami/redirect-web#63] [Questions] [answered]
I would be happy to get the option to use it in different browser, not only in Safari browser

**Comment by mshibanami** [comment]
Hi @0nhub, unfortunately, there is no milestone for supporting other browsers at the moment. The main part of this extension is written in Swift and relies heavily on Apple’s technologies, making it challenging to port to other browsers.

When I created this extension, I didn’t initially consider portability because my primary motivation was addressing the lack of the [Redirector](https://einaregilsson.com/redirector/) extension for Safari, and I often suggest users try Redirector for other browsers. Since its initial release, Redirect Web has supported exporting rules in the Redirector format. You can find more details here:  
https://mshibanami.github.io/redirect-web/export-or-import-rules  

That said, I understand the demand for Redirect Web on other browsers, as Redirector doesn't offer features like iCloud Sync, Open in App, App Lock, DNR, etc.

By the way, there is an ongoing issue about this topic in https://github.com/mshibanami/redirect-web/issues/20. Feel free to continue the discussion there so that it becomes the single source of truth for other users.

**Reply by mshibanami** [reply, marked as answer]
Hi @0nhub! I recently released version 8, which now supports all major browsers! See more details [here](https://mshibanami.github.io/redirect-web/release-notes/8.0.0/).

After releasing version 8, I fixed many bugs, and I think it's pretty stable now. I hope you'll like it. 

Your feedback encouraged me to implement it. Thanks so much! 😄

**Reply by Timodej** [reply]
I always hate to be that guy that asks “could you add a function specifically for my usecase”, but here I am.

I’ve been trying to move away from anything google. Therefore, I’ve switched my searches to Kagi years ago. I’m also using their (swift based) Orion browser ever since. 

This should be a relative easy addition, but I’d understand if you didn’t have the time. 

Thanks in advance. 
Timo

**Reply by mshibanami** [reply]
Hi @Timodej. Your feedback is very helpful!

RedirectWeb for macOS **technically** works on Orion, although it seems unstable due to Orion's implementation of web extension support. I think it will improve in the future. At least I'm surprised RedirectWeb works on Orion now. It didn't work at all when I tried it last year. They're definitely improving.


https://github.com/user-attachments/assets/f9607390-59e3-4997-8c5d-c9df03b5d6c1

So, I think what you meant is Orion for iOS. Unfortunately, supporting Orion for iOS would be even harder than supporting Windows/Linux. This is because the RedirectWeb extension needs to communicate with the RedirectWeb app to get the redirection rules, etc., which is called [Native Messaging](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Native_messaging). However, Orion for iOS doesn't support Native Messaging. So, to support it, I would need to migrate all the app's features to the extension, which would be quite a lot of work. 

That said, I haven't given up on the idea. I'll update you whenever I have anything to share!
